ui/app/styles/core/buttons.scss

```diff
@@ -332,3 +332,72 @@ a.button.disabled {
     cursor: not-allowed;
   }
 }
+
+// Modification to HDS copy button component to match current UI
+.hds-copy-button {
+  &.icon-only {
+    background: none;
+    border: none;
+    box-shadow: none;
+    min-width: $spacing-l;
+    padding: 0;
+  }
+
+  &.white-icon svg {
+    color: $white;
+  }
+
+  &.hidden-icon svg {
+    color: transparent;
+  }
+
+  &.size-7 div {
+    font-size: $size-7;
+  }
```

Comment:
Can't use `is-size-7` helper class here because the font-size needs to be applied to the child `div` in the copy button